data/transactions/logic/tracer_test.go

```diff
@@ -14,94 +14,69 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
 
-package logic
+package logic_test
```

Comment:
By changing to the `logic_test` package, we can now import the `mocktracer` package without circular import issues!